easybuild/easyconfigs/i/ImageJ/ImageJ-1.51a.eb

```diff
@@ -0,0 +1,32 @@
+# Contribution from the Crick HPC team
+# uploaded by J. Sassmannshausen
+
+easyblock = 'PackedBinary'
+
+name = 'ImageJ'
+version = '1.51a'
+
+homepage = 'https://imagej.nih.gov/ij'
+description = "Image Processing and Analysis in Java"
+
+toolchain = {'name': 'dummy', 'version': ''}
+
+source_urls = ['https://imagej.nih.gov/ij/download/src/']
+sources = ['ij%(version_major)s%(version_minor)s-src.zip']
+
+dependencies = [
+    ('Java', '1.8.0_202'),
```

Comment:
@sassy-crick Please change this to use the `Java` 1.8 wrapper, now that #7547 is merged?

Reply:
@boegel done. I think I forgot to upload that last week.